SQL.js

```diff
@@ -35,6 +35,20 @@ class SqlStatement {
     return new SqlStatement(result.strings, result.values)
   }
 
+  /**
+   * A function that accepts an array of objects and a mapper function
+   * It returns a clean SQL format using the object properties defined in the mapper function
+   */
+  map (array, mapFunc) {
+    if ((mapFunc instanceof Function) && array && array.length > 0) {
```

Comment:
```suggestion
    if ((mapFunc instanceof Function) && array?.length) {
```

Reply:
Yeah, I am getting Unexpected token '.' most probably because of the version of node that I am using

Reply:
I have updated the node version and now it works